files/config/default/etc/workflows/uct-include-process-for-empty-venue.xml

```diff
@@ -60,6 +60,43 @@
       </configurations>
     </operation>
 
+    <!-- Load workflow properties from metadata: update some workflow properties -->
+    <operation
+      id="execute-once"
+      if="${empty_venue}"
+      max-attempts="3"
+      retry-strategy="hold"
+      fail-on-error="true"
+      exception-handler-workflow="partial-error"
+      description="Load workflow properties from settings in series">
+      <configurations>
+        <configuration key="exec">/opt/opencast/wfexec/get_organizer_details.pl</configuration>
+        <configuration key="load">0.1</configuration>
+        <configuration key="params">#{id} #{out}</configuration>
+        <configuration key="output-filename">series_organizer.txt</configuration>
+        <configuration key="set-workflow-properties">true</configuration>
+        <configuration key="expected-type">Attachment</configuration>
+      </configurations>
+    </operation>
+
+    <operation
+      id="send-email"
+      fail-on-error="true"
+      if="${organizer_email_valid} AND ${get_organizer_success} AND ${empty_venue}"
+      exception-handler-workflow="partial-error"
+      description="Sending empty venue email to organizer and notification list">
+      <configurations>
+        <!-- <configuration key="to">${organizer_email}</configuration>
```

Comment:
Uncomment and remove test addresses. Include BCC if you want to know what's being emailed out.